assets/javascripts/script.js

```diff
@@ -76,9 +76,11 @@ $(function() {
         var valueSelected = this.value;
         if (valueSelected !== '') {
             $('#revs').addClass('disabled');
+            $('#revs').attr('title', 'Please switch to empty branch before choosing a revision').attr('rel', 'tooltip');
```

Comment:
Not sure how to translate JS label, but a translated label would be good as well.

Reply:
Have searched how to do that for a while, a note for myself: https://ithelp.ithome.com.tw/articles/10188311